pages/event-resources.jsx

```diff
@@ -0,0 +1,407 @@
+var React = require('react');
+var ImageTag = require('../components/imagetag.jsx');
+
+var LinksContainer = React.createClass({
+  render: function() {
+    return (
+      <div className="row">{this.props.children}</div>
+    );
+  }
+});
+var PageLinker = React.createClass({
+  render: function() {
+    return (
+      <div className="page-linker col-sm-3 col-md-3 col-lg-3">
+        <a href={this.props.href}>{this.props.head}</a>
+        {this.props.children}
+      </div>
+    );
+  }
+});
+var EventItem = React.createClass({
+  render: function() {
+    return (
+      <div className="event-item">
+        <div className="event-icon">
+          <div className="participants-number">{this.props.participants}</div>
+          <div className="participants-label">PARTICIPANTS</div>
+        </div>
+        <div className="event-content">
+          <div className="event-item-header">{this.props.head}</div>
+          <div className="event-item-subheader">{this.props.subHead}</div>
+          <p className="event-item-content">{this.props.content}</p>
+        </div>
+      </div>
+    );
+  }
+});
+var LogoAsset = React.createClass({
+  render: function() {
+    return (
+      <div className="logo-asset-container col-sm-4 col-md-4 col-lg-3">
+        <div className="logo-asset-header">{this.props.head}</div>
+        <ImageTag className="logo-asset-img"
+        width={200} height={200}
+        src1x={this.props.src1x} src2x={this.props.src2x}
+        alt={this.props.alt}/>
+        <div className="logo-asset-hover">
+          <div className="logo-asset-center">{this.props.children}</div>
+        </div>
+      </div>
+    );
+  }
+});
+var LogoAssetLink = React.createClass({
+  render: function() {
+    return (
+      <a className="logo-asset-link" href={this.props.href}><span className="ion ion-ios-copy"></span>{this.props.children}</a>
+    );
+  }
+});
+var RemixLink = React.createClass({
+  render: function() {
+    return (
+      <a className="remix-link" href={this.props.href}>Remix</a>
+    );
+  }
+});
+var Tabulator = React.createClass({
```

Comment:
Lol. I mentioned printing because I know it's an issue for other parts of the site, like teaching activities (see #94), and I figured that since this page has so much content, some people might want to print it out for handy reference too, but @LauraHilliger is a better judge than me.

From a technical standpoint this should be pretty easy though, just do a [print media query](http://www.joshuawinn.com/css-print-media-query/) and give it the same styling as your no-js rule.